packages/store-sync/src/createStoreSync.ts

```diff
@@ -215,13 +238,21 @@ export async function createStoreSync({
           ? bigIntMax(range.startBlock, lastBlockNumberProcessed + 1n)
           : range.startBlock,
         toBlock: range.endBlock,
-        storageAdapter,
         logFilter,
+        storageAdapter,
       });
 
       return from(storedBlocks);
     }),
-    tap(({ blockNumber, logs }) => {
+  );
+
+  const storedBlock$ = storedIndexerLogs$.pipe(
+    catchError((e) => {
+      debug("error streaming logs from indexer:", e);
+      debug("falling back to streaming logs from RPC");
+      return storedRpcLogs$;
+    }),
```

Comment:
have we tested the fallback behavior here?

Reply:
The fallback behavior already kicks in if no indexer is provided or the existing indexer doesn't support this API, so in that sense it's covered by the existing e2e tests